arangod/GeneralServer/RestHandler.cpp

```diff
@@ -175,87 +153,45 @@ bool RestHandler::forwardRequest() {
     params.append(StringUtils::urlEncode(i.second));
   }
 
-  auto cc = ClusterComm::instance();
-  if (cc == nullptr) {
-    // nullptr happens only during controlled shutdown
-    generateError(rest::ResponseCode::SERVICE_UNAVAILABLE,
-                  TRI_ERROR_SHUTTING_DOWN, "shutting down server");
-    return true;
-  }
-
-  std::unique_ptr<ClusterCommResult> res;
-  if (!useVst) {
-    HttpRequest* httpRequest = dynamic_cast<HttpRequest*>(_request.get());
-    if (httpRequest == nullptr) {
-      THROW_ARANGO_EXCEPTION_MESSAGE(TRI_ERROR_INTERNAL,
-                                     "invalid request type");
+  auto requestType =
+      fuerte::from_string(GeneralRequest::translateMethod(_request->requestType()));
+  auto payload = _request->toVelocyPackBuilderPtr()->steal();
+  auto future = network::sendRequestRetry("server:" + serverId, requestType,
```

Comment:
sendRequestRetry does more than `ClusterComm::syncRequest`, is that intended ?

Reply:
I may have misunderstood the nature of the method. Does it generally robustify the request procedure, or does it have a more specific use case? Would `network::methods::sendRequest` be a more direct conversion that you'd recommend?

Reply:
yes that would be the direct equivalent

Reply:
Fixed.